florian.md

```diff
@@ -0,0 +1,8 @@
+# Heading 1
+
+## Heading 2
+
+- List
+- Listitem 2
+
+[ ] checkbox
```

Comment:
This looks great! The checkbox might work better if there's a `- ` in front, so it's like `- [ ] checkbox` instead of `[  ] checkbox`. 👍